Burnr has a bunch of missing await errors

When running `yarn dev` in burnr:

```bash
src/App.tsx:43:16 - error TS2801: This condition will always return true since this 'Promise<import("/home/raoul/code/paritytech/substrate-connect/node_modules/@polkadot/api/promise/index").ApiPromise>' is always defined.

43     if (api && api.isReady) {
                  ~~~~~~~~~~~

  src/App.tsx:43:16
    43     if (api && api.isReady) {
                      ~~~~~~~~~~~
    Did you forget to use 'await'?

src/components/NodeSelector.tsx:108:95 - error TS2801: This condition will always return true since this 'Promise<import("/home/raoul/code/paritytech/substrate-connect/node_modules/@polkadot/api/promise/index").ApiPromise>' is always defined.

108             <FiberManualRecordIcon style={{ fontSize: '16px', marginRight: 4 }} color={api && api.isReady ? 'primary' : 'error'} />
                                                                                                  ~~~~~~~~~~~

  src/components/NodeSelector.tsx:108:95
    108             <FiberManualRecordIcon style={{ fontSize: '16px', marginRight: 4 }} color={api && api.isReady ? 'primary' : 'error'} />
                                                                                                      ~~~~~~~~~~~
    Did you forget to use 'await'?

src/hooks/api/useApiCreate.ts:8:27 - error TS2307: Cannot find module '@substrate/connect' or its corresponding type declarations.

8 import { Detector }  from '@substrate/connect';
                            ~~~~~~~~~~~~~~~~~~~~


Found 3 errors.
```

--- a/projects/burnr/src/App.tsx
+++ b/projects/burnr/src/App.tsx
@@ -40,16 +40,20 @@ const App: React.FunctionComponent<Props> = ({ className = '' }: Props) => {
   const [loader, setLoader] = useState(true)
 
   useEffect((): void => {
-    if (api && api.isReady) {
-      if (!localStorageAccount) {
-        const userTmp = createLocalStorageAccount();
-        setLocalStorageAccount(JSON.stringify(userTmp));
-        setCurrentAccount(userTmp);
-      } else {
-        setCurrentAccount(JSON.parse(localStorageAccount));
-      }
-      setLoader(false)
+    const callSetters = async () => {
+      if (await api.isReady) {
+        if (!localStorageAccount) {
+          const userTmp = createLocalStorageAccount();
+          setLocalStorageAccount(JSON.stringify(userTmp));
+          setCurrentAccount(userTmp);
+        } else {
+          setCurrentAccount(JSON.parse(localStorageAccount));
+        }
+        setLoader(false);
+      }  
     }
+
+    api && callSetters();
   }, [localStorageAccount, setLocalStorageAccount, api]);
 
   return (

--- a/projects/burnr/src/components/NodeSelector.tsx
+++ b/projects/burnr/src/components/NodeSelector.tsx
@@ -1,8 +1,10 @@
-import React, { useState, ChangeEvent } from 'react';
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import React, { useState, ChangeEvent, useEffect } from 'react';
 
 import { createStyles, makeStyles, Theme  } from '@material-ui/core/styles';
 import { InputBase, ClickAwayListener,  Typography, Box } from '@material-ui/core';
 import Autocomplete, { AutocompleteCloseReason } from '@material-ui/lab/Autocomplete';
+import { ApiPromise } from '@polkadot/api';
 
 import { ALL_PROVIDERS } from '../utils/constants';
 import { useLocalStorage, useApi } from '../hooks';
@@ -70,6 +72,8 @@ const options = Object.entries(ALL_PROVIDERS).map(
   )
 ).sort((a,b) => (a.network > b.network) ? 1 : ((b.network > a.network) ? -1 : 0));
 
+type colorType = 'inherit' | 'primary' | 'secondary' | 'action' | 'disabled' | 'error';
+
 export default function NodeSelector(): React.ReactElement {
   const classes = useStyles();
   const api = useApi();
@@ -77,6 +81,7 @@ export default function NodeSelector(): React.ReactElement {
   const endpointName = localEndpoint || 'Westend-WsProvider';
   const [provider, setProvider] = useState<string>(ALL_PROVIDERS[endpointName]?.id);
   const [open, setOpen] = useState<boolean>(false);
+  const [fiberColor, setFiberColor] = useState<colorType>('error');
 
   const toggleOpen = () => {
     setOpen(!open);
@@ -99,13 +104,24 @@ export default function NodeSelector(): React.ReactElement {
     // setChain(REMOTE_PROVIDERS[selectedEndpoint].network);
   };
 
+  useEffect(() => {
+    const getColor = async (api: ApiPromise) => {
+      if (api && await api.isReady) {
+        setFiberColor('primary');
+      }
+      setFiberColor('error');
+    }
+
+    api && getColor(api);
+  }, [api]);
+
   return (
     <ClickAwayListener onClickAway={handleClose}>    
       <div className={classes.nodeSelectorWrap}>
         <div className={`${classes.nodeSelectorInner} ${open ? 'open' : ''}`}>
 
           <Box display='flex' alignItems='center' pt={1.5} pb={1.5} pl={0.5} pr={0.5} onClick={toggleOpen}>
-            <FiberManualRecordIcon style={{ fontSize: '16px', marginRight: 4 }} color={api && api.isReady ? 'primary' : 'error'} />
+            <FiberManualRecordIcon style={{ fontSize: '16px', marginRight: 4 }} color={fiberColor} />
             <Box width='100%'>
               <Typography variant='h4'>{ ALL_PROVIDERS[provider]?.network }</Typography>
               <Typography variant='body2' color='textSecondary'>{ALL_PROVIDERS[provider]?.client} client</Typography>

--- a/projects/burnr/src/components/NodeSelector.tsx
+++ b/projects/burnr/src/components/NodeSelector.tsx
@@ -109,7 +109,6 @@ export default function NodeSelector(): React.ReactElement {
       if (api && await api.isReady) {
         setFiberColor('primary');
       }
-      setFiberColor('error');
     }
 
     api && getColor(api);

